### DISCUSSION TITLE
Mac install issue

### DISCUSSION THREAD

**Discussion by jerrytom1986** [darekkay/evaluatory#11] [Q&A] [answered]
Hi,

I trying to use evaluators on MacOS/Terminal and I still stuck in installation playwright, I am not able to run it? 

× browserType.launch: Executable doesn't exist at /Users/pz/Library/Caches/ms-playwright/chromium-1091/chrome-mac/Chromium.app/Contents/MacOS/Chromium
╔═════════════════════════════════════════════════════════════════════════╗
║ Looks like Playwright Test or Playwright was just installed or updated. ║
║ Please run the following command to download new browsers:              ║
║                                                                         ║
║     npx playwright install                                              ║
║                                                                         ║
║ <3 Playwright Team                                                      ║
╚═════════════════════════════════════════════════════════════════════════╝
i Saved results to: /Users/pz/evaluatory-results
× Finished evaluation: 1 issues found.

**Comment by darekkay** [comment]
The issue should be resolved with Evaluatory `4.1.0`.

~With the latest playwright version, the `playwright` dependency is no longer installed locally/automatically. You need to follow the steps from the message `npx playwright install`~

**Reply by jerrytom1986** [reply]
I did this, and the same issue. over and over.

**Reply by jerrytom1986** [reply]
% npx playwright --version
Version 1.39.0

**Reply by azat-io** [reply]
I have the same problem

**Reply by darekkay** [reply]
Thank you, I can reproduce the issue on macOS (it works fine on Windows and Linux). I'll have a look.

**Reply by darekkay** [reply, marked as answer]
The issue should be resolved with the latest release `4.1.0`. I've added the browsers as dependencies as documented [here](https://playwright.dev/docs/library#key-differences).

Somehow my `npx evaluatory@4.1.0 URL` didn't work immediately. I've cleared the `~/.npm/_npx` folder and re-installed the package globally via `npm install -g evaluatory@4.1.0`. Now both `evaluatory URL` and `npx evaluatory URL` work as expected on macOS.

Let me know if the issue persist for you (even after clearing the `_npx` folder).

NOTE: I did **not** have to run `npx playwright install`.

**Reply by azat-io** [reply]
Awesome tool. Thank you!

**Reply by jerrytom1986** [reply]
Drinks on me! It works!
